configure.py

```diff
@@ -652,6 +652,8 @@ def find_headers(repodir, excluded_dirs):
                         help="LLVM-specific inline threshold compilation parameter")
 arg_parser.add_argument('--list-artifacts', dest='list_artifacts', action='store_true', default=False,
                         help='List all available build artifacts, that can be passed to --with')
+arg_parser.add_argument('--date-stamp','-d', dest='date_stamp', type=str,
+                        help='Set datestamp for SCYLLA-VERSION-GEN')
```

Comment:
',' wants a space after it. But better to drop -d, this parameter has a big effect and we want to make sure whoever uses it  is aware of what they do.

Reply:
Removed